src/cryptography/hazmat/primitives/serialization.py

```diff
@@ -178,6 +186,10 @@ class PublicFormat(Enum):
     OpenSSH = "OpenSSH"
 
 
+class ParameterFormat(Enum):
+    DHParameter = "PKCS3"
```

Comment:
I am waffling on whether we should principally call it DHParameter or PKCS3. I guess the docs already say it's a DHParameter structure as defined in PKCS3 so I guess that's good enough. Do you have an opinion @palaviv?